spark/src/test/scala/org/apache/spark/sql/delta/InCommitTimestampSuite.scala

```diff
@@ -179,6 +179,8 @@ class InCommitTimestampSuite
   }
 
   test("Missing CommitInfo should result in a DELTA_MISSING_COMMIT_INFO exception") {
+    // Make sure that we don't retrieve the time from the CRC.
+    withSQLConf(DeltaSQLConf.DELTA_WRITE_CHECKSUM_ENABLED.key -> "false") {
     withTempDir { tempDir =>
```

Comment:
odd (lack of) indentation here?

(several more below)

Reply:
Fixed the indentation. I wanted to keep the diff cleaner but given that the content itself has been reviewed, that should not matter anymore

Reply:
FYI - Indentation changes are easily hidden:
![image](https://github.com/user-attachments/assets/b362070f-94a2-4144-a252-1a210f17dc5b)

Reply:
... which gives e.g.
![image](https://github.com/user-attachments/assets/53dd35ac-2de9-4c4c-b226-9ad89d8fd108)